tests/test_sortedLinkedList.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+from ContractsFixture import ContractsFixture
+from ethereum.tester import TransactionFailed
+from pytest import raises
+
+def test_sortedLinkedList():
```

Comment:
Recommend breaking this up into a large number of smaller tests rather than one monolithic test.  Such a change would allow each test to be executed in isolation (useful for debugging), as well as make the test easier to grok since we each test will start with a clean slate.

In order to avoid the performance hit of this, you can create a test fixture that uploads the contract and then takes a snapshot, then restores the snapshot for each test:

```
@fixture(scope = 'session')
def mySnapshot(contractsFixture):
    contractsFixture.upload('../src/libraries/sortedLinkedList.se')
    contractsFixture.state.mine(1)
    return contractsFixture.state.snapshot()

@fixture
def myImprovedContractsFixture(contractsFixture, mySnapshot):
    contractsFixture.state.revert(mySnapshot)
    return contractsFixture

def test_something(myImprovedContractsFixture):
    linkedList = myImprovedContractsFixture.contracts['sortedLinkedList']
    ...
```